### PR TITLE
Release v0.10.0

### PR DESCRIPTION
**Pull request by yoyama**
Release v0.10.0

### CODE REVIEW

File: digdag-docs/src/releases/release-0.10.0.rst [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+* Command Executor SPI has been updated and AWS ECS Command Executor has been introduced. [#835 #1127]
+* Java11 has been supported. (experimental) [#1256 #1361]
+* Graaljs has been supported for ConfigEvalEngine and performance dramatically improved. [#1256 #1361]
+* AccessController SPI has been introduced.  (experimental) [#936]
+* Metrics improvements. (experimental) [#1146]
+* Authentication configuration was changed. [#1324]
+* Abolished JWT authentication. [#1324]
+* Some API have been abolished and changed. [#1109 #1110 #1114 #1115 #1224 #1288 #1398 #1401]
+* `_parallel` option supports a number to control number of parallels. [#1413]
+* The maximum number of tasks, attempts are configurable. [#1430]
```
Comment (komamitsu) [inline digdag-docs/src/releases/release-0.10.0.rst:15]: [minor] `The maximum numbers of tasks and attempts are configurable` might be better?

File: digdag-docs/src/releases/release-0.10.0.rst [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+* Upgrade Gradle to 6.3 [#1391]
+
+Fixed Issues
+------------
+* System environment variables are not passed in except Simple Command Executor. [#1101]
+* Overwriting runtime parameters has been prohibited. [#1168]
+* Group retry improvements. [#1184]
+* `http>` operator improvements. [#1197]
+* `s3_wait>` operator improvements. [#1280]
+* `py> operator` improvements. [#1477]
```
Comment (komamitsu) [inline digdag-docs/src/releases/release-0.10.0.rst:25]: `py>` operator ?

File: digdag-docs/src/releases/release-0.10.0.rst (hunk 1 above)
Comment (yoyama) [reply digdag-docs/src/releases/release-0.10.0.rst:15]: Thanks, I will fix it.

File: digdag-docs/src/releases/release-0.10.0.rst (hunk 2 above)
Comment (yoyama) [reply digdag-docs/src/releases/release-0.10.0.rst:25]: Thanks. I will fix it.

**Review by komamitsu — APPROVED** [review]
👍

**Pull request merged**